src/main/java/be/uantwerpen/scicraft/util/NuclidesTable.java

```diff
@@ -0,0 +1,566 @@
+//        addNuclidesTableEntry(19, 23, "Potassium", "K", "Unstable", null, 1);
+//        addNuclidesTableEntry(19, 24, "Potassium", "K", "Unstable", null, 2);
+//        addNuclidesTableEntry(19, 25, "Potassium", "K", "Unstable", null, 3);
+//        addNuclidesTableEntry(19, 26, "Potassium", "K", "Unstable", null, 4);
+//        addNuclidesTableEntry(19, 27, "Potassium", "K", "Unstable", null, 5);
+//        addNuclidesTableEntry(19, 28, "Potassium", "K", "Unstable", null, 6);
+//        addNuclidesTableEntry(19, 29, "Potassium", "K", "Unstable", null, 7);
+//        addNuclidesTableEntry(19, 30, "Potassium", "K", "Unstable", null, 8);
+//        addNuclidesTableEntry(19, 31, "Potassium", "K", "Unstable", null, 9);
+//        addNuclidesTableEntry(19, 32, "Potassium", "K", "Unstable", null, 10);
+//        addNuclidesTableEntry(19, 33, "Potassium", "K", "Unstable", null, 11);
+//        addNuclidesTableEntry(19, 34, "Potassium", "K", "Unstable", null, 12);
+//        addNuclidesTableEntry(19, 35, "Potassium", "K", "Unstable", null, 13);
+//        addNuclidesTableEntry(19, 36, "Potassium", "K", "Unstable", null, 14);
+//        addNuclidesTableEntry(19, 37, "Potassium", "K", "Unstable", null, 15);
+
+        // define electron shells map
+        shells.put(1, 2);
+        shells.put(2, 8);
+        shells.put(3, 18);
+        shells.put(4, 32);
+        shells.put(5, 50);
+        shells.put(6, 72);
+        shells.put(7, 98);
+        shells.put(8, 128);
+        shells.put(9, 162);
+
+    }
+
+    private static Map<String, NucleusState> readCSV() {
+
+        Map<String, NucleusState> _nuclidesTable = new HashMap<>();
+
+        String line = "";
+        String splitBy = ",";
+        Set<String> decays = new HashSet<>();
+        try {
+            //parsing a CSV file into BufferedReader class constructor
+//            URL url = NuclidesTable.class.getResource("nndc_nudat_nuclidetable.csv");
+//            File myFile = new File("nndc_nudat_nuclidestable.csv");
+//            InputStream inputStream = NuclidesTable.class.getClassLoader().getResourceAsStream("../src/main/java/be/uantwerpen/scicraft/util/nndc_nudat_nuclidestable.csv");
+//            BufferedReader br = new BufferedReader(new InputStreamReader(inputStream));
+            BufferedReader br = new BufferedReader(new FileReader("../src/main/resources/data/nuclidesTable/nndc_nudat_data_export.csv"));
+            int it = 0;
+
+            while ((line = br.readLine()) != null) {
+                if (it == 0) {
+                    it++; continue;
+                }
+                String[] nuclideInfo = line.split(splitBy); // use comma as separator (see .csv file)
+                String symbol = nuclideInfo[2].replaceAll("[0-9]", "");
+                Atom atom = Atom.getBySymbol(symbol);
+                String atomName = "";
+                Item atomItem = null;
+                if (atom != null) {
+//                    atomName = atom.getItem().getName().toString();
```

Comment:
Remove commented out code if it is not needed anymore

Reply:
This commented code might be needed, since the current approach is not necessarily the right one.